.github/workflows/obs-helper.yaml

```diff
@@ -0,0 +1,22 @@
+---
+# yamllint disable rule:line-length
+name: OBS Helper
+
+on: [pull_request_target]
+
+jobs:
+  obs-url:
+    name: Report OBS Url
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - uses: actions/checkout@v4
+      - name: Comment
+        # Only run once on pull request creation
+        if: github.run_number == 1
```

Comment:
Could we also make this depend on whether the OBS hook failed?

Reply:
Although I'm not sure why we can't fix the original OBS check to always show the URL. It **seems** like that should be trivial as the URL is predictable.

Reply:
> Could we also make this depend on whether the OBS hook failed?

I wasn't successful to query webhook deliveries via command line so far.
https://docs.github.com/en/rest/apps/webhooks?apiVersion=2022-11-28

Reply:
> Although I'm not sure why we can't fix the original OBS check to always show the URL. It **seems** like that should be trivial as the URL is predictable.

Wanna make a PR? https://github.com/openSUSE/open-build-service/blob/7d122807c183ef66a4b8ad92f03de98ca06a2a2b/src/api/app/services/github_status_reporter.rb#L49
:)

Reply:
But I guess it might not be trivial. I think it's possible to configure several different workflows in such a file, so there is not only one URL with the results.
And as far as I know, already reporting the various states as pending from the beginning is not possible currently due to a limitation in OBS.